README.md

```diff
@@ -1,14 +1,33 @@
 # RISE
 
-**Reveal.js - Jupyter/IPython Slideshow Extension**, also known as *live_reveal*.
-Quickly turn your Jupyter Notebooks into a live presentation.
+RISE allows you to instantly turn your Jupyter Notebooks into a
```

Comment:
I would prefer to keep the word "live", I have found people understanding better the RISE idea when I told them it is a "live" slideshow where they can actually execute code.

Reply:
fine - will be in today's commit

Reply:
Thx!